### PR TITLE
Migrate to GH Actions

### LINKED ISSUE #8: global: migration to github-actions from travis
closes #9

### PR DESCRIPTION
**Pull request by avivace**
This PR was prepared by an [automated action](https://github.com/inveniosoftware/automation-tools/tree/master/automation_tools/scripts/ga-migration). Closes #8

### CODE REVIEW

**Comment by mvidalgarcia** [comment]
This PR is missing to be reviewed since its issue wasn't attached to the Kanban board.
TODO: Change commit message to follow Invenio guidelines.

File: run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+python -m check_manifest --ignore ".*-requirements.txt"
+python -m sphinx.cmd.build -qnNW docs docs/_build/html
+python -m pytest
+python -m sphinx.cmd.build -qnNW -b doctest docs docs/_build/doctest
```
Comment (ParthS007) [inline run-tests.sh:19]: Minor: Add empty line

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+name: CI
```
Comment (ParthS007) [inline .github/workflows/tests.yml]: Update with latest `tests.yml`

**Review by ParthS007 — COMMENTED** [review]
Some Minor comments and Add `Github CI` badge in `Readme.MD`

File: .github/workflows/tests.yml (hunk 2 above)
Comment (ParthS007) [inline .github/workflows/tests.yml]: Major: Replace this `js-tests.yml`. It is not a python repository

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Review by ParthS007 — APPROVED** [review]

File: run-tests.sh (hunk 1 above)
Comment (mvidalgarcia) [reply run-tests.sh:19]: Minor: Was this fixed?

**Comment by avivace** [comment]
LGTM, thanks @ParthS007

**Pull request merged**